### DISCUSSION TITLE
Cant login to the desktop app. It looks like something went wrong. Please retry after some time. If the error persists, please contact our support team.

### DISCUSSION THREAD

**Discussion by student2374** [ente-io/ente#3232] [Q&A]
![2024-09-11 13_24_00-Ente Auth](https://github.com/user-attachments/assets/b9a83a74-6374-474b-b8f6-e86d04d9f3d6)

Hi, I am getting this message when I try to login to the desktop app:
It looks like something went wrong. Please retry after some time. If the error persists, please contact our support team.

I am using windows 11 and have the latest version of Ente Installed. (3.1.3)

I have tried to uninstall and reinstall multiple times but getting the same message

**Comment by AntsyBoi** [comment]
Submit this as a bug report here:
https://github.com/ente-io/ente/issues/new?assignees=&labels=triage&projects=&template=bug_report.yml
